third_party/amd/lib/TritonAMDGPUToLLVM/LoadStoreOpToLLVM.cpp

```diff
@@ -774,27 +774,37 @@ struct AtomicRMWOpConversion
     // vec = 1, numElements = 1 for scalar
     auto vec = getVectorSize(ptr);
     int numElems = 1;
+    bool forceF16packing = false;
```

Comment:
Maybe renaming it as `useDppForPackedF16` to be clear? Also can you add a comment regarding what this is for. Basically embed the commit message you have in the pull request. And also mention that "this way enables us to use half the active threads committing atomic requests thus reduce contention and improve atomics performance".